app/controllers/api/v1/api_controller.rb

```diff
@@ -43,6 +43,18 @@ def render_objects_with_cache(object, pid)
       end
     end
 
+    # nil.to_i = 0
+    def page
+      @page = params['page'].to_i
+      (@page == 0) ? 1 : @page
```

Comment:
instead of `== 0` we can use `.zero?` method here

Reply:
yeah, sure @namrataukirde , will do that. Thanks :)